ntfs_fr.md

```diff
@@ -22,7 +22,7 @@ Les données sont formatées de la manière suivante :
 * Le format de l'heure est **HH:MM:SS**
```

Comment:
Should you add a link to the new documentation `daylight_saving.md` here? I know it doesn't affect the format itself, but related concepts should be taken into consideration when dealing with time fields.

Reply:
I've decided not to, keeping the spec as simple to read as possible. References are provided in `*_timezone` fields, I think it's good enought like this. Is it ok with you?